dream_layer_backend/dream_layer.py

```
@@ -614,11 +599,79 @@ def get_controlnet_models_endpoint():
             "message": f"Failed to fetch ControlNet models: {str(e)}"
         }), 500
 
+@app.route('/api/save-single-image', methods=['POST'])
+def save_single_image():
+    """
+    Save single generated image to server storage for persistence
+    """
+    try:
+        print(f"🔄 Single image save request received on port 5002")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)